README.md

```diff
@@ -1,48 +1,11 @@
 # k8s-rdma-sriov-dev-plugin
 (https://hub.docker.com/r/rdma/k8s-rdma-sriov-dev-plugin)
 
-This is simple rdma device plugin that support IB and RoCE SRIOV vHCA and HCA.
-This also support DPDK applications for Mellanox NICs.
+This is simple rdma device plugin that support IB and RoCE HCA.
 This plugin runs as daemonset.
 Its container image is available at rdma/k8s-rdma-sriov-dev-plugin.
```

Comment:
update to k8s-rdma-sriov-dev-plugin to k8s-rdma-shared-dev-plugin (can you also rename the repo to k8s-rdma-shared-dev-plugin